CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 0.24.0 / 2018-10-10
+
+* [CHANGE] Default to Node Exporter v0.16.0 (#1812)
+* [CHANGE] Update to Go 1.11 (#1855)
+* [CHANGE] Default to Prometheus v2.4.3 (#1929) (#1983)
+* [CHANGE] Default to Thanos v0.1.0 (#1954)
+* [CHANGE] Overhaul metrics while adding triggerBy metric for Alertmanager (#1984)
```

Comment:
Can you add a description just below the release heading that describes what users need to change on their end in regard to the metrics changes?

Reply:
I've done that. Ready to be reviewed. :+1: